### ISSUE TITLE
Cannot attack players

### ISSUE THREAD

**Issue by IsXGeo** [ScreamingSandals/BedWars#94]
#### Describe the bug
Even with respawn protection disabled, players cannot attack enemies for a few seconds after respawn. This makes last second bed defense entirely impossible.

There are no crash logs for this.

If this is a feature and not a bug, could I request a way to disable it?

#### Server and plugin version
Spigot version: Paper-MC 230
BedWars version: 2.4 SNAPSHOT-413

#### Installed plugins on server (can be obtained by `/pl`)
AdvancedBan, Bedwars, CommandPanels, CoreProtect, Essentials, EssentialsAntiBuild, EssentialsChat, EssentialsProtect, EssentialsSpawn, LuckPerms, Matrix, PlaceholderAPI, ProtocolLib, Vault, WorldEdit, WorldGuard

#### Configuration (of config.yml and shop.yml)
Config - https://pastebin.com/GWQZJ6bK
Shop - https://pastebin.com/t6QxRfpe

**Comment by Misat11** [comment]
that's not bug, just change respawn protection time in config.yml :) under respawn-protection section

**Comment by IsXGeo** [comment]
I have this set to disabled aswell as zero however there are still a few seconds of spawn protection and the inability to hit players. The inability to hit others also is only active half the time the spawn protection is

**Comment by Misat11** [comment]
set respawn.protection-enabled to false

**Comment by IsXGeo** [comment]
I have done so yet the problem persists

**Comment by Misat11** [comment]
for me it's working correctly. Are you sure you have disabled respawn protection and not respawn cooldown? Or are you sure this isn't bug caused by another plugin?

**Comment by IsXGeo** [comment]
This is how it is in my config
respawn:
    protection-enabled: false
    protection-time: 0
And my plugin list has not changed from original post
Essentials doesn't seem to have built in spawn protection and world guard isnt used in the vicinity of my maps.
When a player respawns, even with the protection disabled in config they have a few seconds of invincibility and are also unable to attack players. I am referring to the instant they spawn
Apologies for this drawn out issue

**Comment by IsXGeo** [comment]
I have found the answer to this issue. I had assumed that a player "respawned" when they were back in the match. Technically they were already alive once in spectator mode. As such I was looking in the wrong place. The problem was indeed Essentials however nothing to do with respawn protection as much as teleportation invulnerability. I was looking in the wrong place. Sorry to take up your time with this issue.

TLDR; Disable Essentials Teleport Invulnerability to fix this issue.
